### PR TITLE
feat(icons): comment.svg

### PR DESCRIPTION
**Pull request by mhuet**
**What is the problem this PR is trying to solve?**
A comment icon was needed for the new comment feature

**What is the chosen solution to this problem?**
The icon has been created

**Please check if the PR fulfills these requirements**

* [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/icons/src/svg/core/comment.svg [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+  <g>
+    <path d="M13.697561,-4.45520775e-05 C14.3064337,-0.00374844656 14.8917413,0.229036047 15.3238873,0.646856976 C15.7560334,1.0646779 15.9993633,1.63306256 16,2.22616152 L16,9.80357547 C16,11.0423127 14.9692683,12.0629524 13.697561,12.0629524 L6.76243902,12.0629524 L3.58439024,15.5687137 C2.98097561,16.2695719 2.4897561,16.098515 2.49317073,15.1838356 L2.49317073,12.0629524 L2.30243902,12.0629524 C1.03073171,12.0629524 0,11.0423127 0,9.80357547 L0,2.22616152 C0.000636672352,1.63306256 0.243966605,1.0646779 0.676112676,0.646856976 C1.10825875,0.229036047 1.69356634,-0.00374844656 2.30243902,-4.45520775e-05 L13.697561,-4.45520775e-05 Z M5,5 C4.44771525,5 4,5.44771525 4,6 C4,6.55228475 4.44771525,7 5,7 C5.55228475,7 6,6.55228475 6,6 C6,5.44771525 5.55228475,5 5,5 Z M8,5 C7.44771525,5 7,5.44771525 7,6 C7,6.55228475 7.44771525,7 8,7 C8.55228475,7 9,6.55228475 9,6 C9,5.44771525 8.55228475,5 8,5 Z M11,5 C10.4477153,5 10,5.44771525 10,6 C10,6.55228475 10.4477153,7 11,7 C11.5522847,7 12,6.55228475 12,6 C12,5.44771525 11.5522847,5 11,5 Z"/>
```
Comment (frassinier) [inline packages/icons/src/svg/core/comment.svg:3]:
```suggestion
    <path class="ti-foreground" d="M13.697561,-4.45520775e-05 C14.3064337,-0.00374844656 14.8917413,0.229036047 15.3238873,0.646856976 C15.7560334,1.0646779 15.9993633,1.63306256 16,2.22616152 L16,9.80357547 C16,11.0423127 14.9692683,12.0629524 13.697561,12.0629524 L6.76243902,12.0629524 L3.58439024,15.5687137 C2.98097561,16.2695719 2.4897561,16.098515 2.49317073,15.1838356 L2.49317073,12.0629524 L2.30243902,12.0629524 C1.03073171,12.0629524 0,11.0423127 0,9.80357547 L0,2.22616152 C0.000636672352,1.63306256 0.243966605,1.0646779 0.676112676,0.646856976 C1.10825875,0.229036047 1.69356634,-0.00374844656 2.30243902,-4.45520775e-05 L13.697561,-4.45520775e-05 Z M5,5 C4.44771525,5 4,5.44771525 4,6 C4,6.55228475 4.44771525,7 5,7 C5.55228475,7 6,6.55228475 6,6 C6,5.44771525 5.55228475,5 5,5 Z M8,5 C7.44771525,5 7,5.44771525 7,6 C7,6.55228475 7.44771525,7 8,7 C8.55228475,7 9,6.55228475 9,6 C9,5.44771525 8.55228475,5 8,5 Z M11,5 C10.4477153,5 10,5.44771525 10,6 C10,6.55228475 10.4477153,7 11,7 C11.5522847,7 12,6.55228475 12,6 C12,5.44771525 11.5522847,5 11,5 Z"/>
```

**Comment by tlnd-mhuchet** [comment]
Is the failing test really related to the new icon? 🤔

**Comment by frassinier** [comment]
> Is the failing test really related to the new icon? 🤔

no

File: packages/icons/src/svg/core/comment.svg (hunk 1 above)
Comment (vbocquet) [reply packages/icons/src/svg/core/comment.svg:3]: @frassinier Can we close the PR or this change is requested?

Comment (frassinier) [reply packages/icons/src/svg/core/comment.svg:3]: Yes I will do the change

Comment (vbocquet) [reply packages/icons/src/svg/core/comment.svg]: Cool, thx 👍

**Pull request merged**